.gitignore

```diff
@@ -22,3 +22,9 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
```

Comment:
🤔 I see there is now a `.env` in the `.gitignore` file. Did you by any chance create secrets in the course of your changes?

Reply:
Oh... This, supposed to be for another project. 
My bad :(